cmd/postpolicyform.go

```diff
@@ -270,7 +270,7 @@ func checkPostPolicy(formValues http.Header, postPolicyForm PostPolicyForm) erro
 	ignoreKeys := map[string]bool{}
 	for key, value := range formValues {
 		switch {
-		case ignoreKeys[key], key == "Policy", key == "X-Amz-Signature":
+		case ignoreKeys[key], key == "Policy", key == "X-Amz-Signature", key == "Content-Encoding":
 			continue
 		case strings.HasPrefix(key, "X-Amz-Ignore-"):
```

Comment:
use `HasPrefix` for case-insensitive check.

Reply:
the `formValues ` have case-insensitive key already.
```go
			formValues[http.CanonicalHeaderKey(name)] = append(formValues[http.CanonicalHeaderKey(name)], b.String())
```